include/ipmitool/ipmi_sel.h

```diff
@@ -527,6 +527,7 @@ static const struct ipmi_event_sensor_types sensor_specific_event_types[] = {
     { 0x2b, 0x01, 0x15, "Firmware or software change detected, board/FRU replace new ver" },
     { 0x2b, 0x01, 0x16, "Firmware or software change detected, board/FRU replace old ver" },
     { 0x2b, 0x01, 0x17, "Firmware or software change detected, board/FRU HW conf change" },
+    { 0x2b, 0x01, 0xff, "Firmware or software change detected" },
```

Comment:
This `0xff` value doesn't seem to be valid as per IPMI 2.0 specification.
The `unspecified` Event Data 2 byte value must be `0x00` (that is listed above), not `0xff`.
I may agree that `ipmitool` shouldn't silently ignore incorrect values, but then I'm curious why only `0xff` gets the honor of being handled among all other out-of-spec values.

I'd propose to alter the actual code, not just the dictionary, to print something like `Invalid sensor event data: XX XX XX` for cases like this.

Reply:
The unspecified Event Data 2 byte value must be 0xff.  Please refer to IPMI 2.0 Table 29-, Event Request Message Event Data Field Contents.

Reply:
@Journey-x, please pay more attention to the spec. Table 29-6 "Event Request Message Event Data Field Contents" specifies field values for "threshold", "discrete", and "OEM" sensors, not for the "Version Change" sensor type, values of Event Data 2 for which are covered by table 42-3 "Sensor Type Codes" and contain no `0xff`.